deployments/kubehound/ui/README.md

```diff
@@ -0,0 +1,41 @@
+# UI
+
+## Jupyter Notebooks
+
+This UI is using Amazon's [Graph Notebook](https://github.com/aws/graph-notebook) tooling to query JanusGraph.
+
+As a first step, we recommend following the [KubeHound DSL 101 notebook](./KubehoundDSL_101.ipynb). You will find details on how to connect, default configuration and how to construct requests there.
+
+## Gremlin tricks
+
+### Cell magic
+
+As stated in the 101 notebook, to launch gremlin queries against the KubeHound graph, you need to start the cell with the jupyter cell magic `%%gremlin`
+
+### Display help
+
+This magic can take a bunch of parameters, you can get the help message by running a cell containing just
+```python
+&&gremlin --help
```

Comment:
```suggestion
%%gremlin --help
```

shouldn't it be `%%` as mentionned above?

Reply:
thanks for the catch